manifests/1.3.0/opensearch-1.3.0.yml

```diff
@@ -34,7 +34,7 @@ components:
       - gradle:dependencies:opensearch.version: opensearch-ml-plugin
   - name: alerting
     repository: https://github.com/opensearch-project/alerting.git
-    ref: main
+    ref: "1.3"
```

Comment:
Thanks for the update here! Could you change the double quote here to single quote like `'1.3'`. We used to have `'` so want to be consistent on this. We will take care of the rest for those already merged. Thanks!

Reply:
Will do. I'll revise this PR now.

Reply:
Never mind, seeing the below comment. Will **not** be revising this PR to use single quotes.